From a4620ec42e00170f5de5692246b88321328a9c0a Mon Sep 17 00:00:00 2001
From: Alberto Bursi <bobafetthotmail@gmail.com>
Date: Sun, 22 Nov 2020 17:54:28 +0100
Subject: [PATCH 3/3] hwinfo: update to version 21.71

update to upstream version 21.71

Signed-off-by: Alberto Bursi <bobafetthotmail@gmail.com>
(cherry picked from commit 7dc7e46d4840ae6905eec995e277553cd96e6639)
---
 utils/hwinfo/Makefile | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/utils/hwinfo/Makefile b/utils/hwinfo/Makefile
index 347d3488d..ffa668b05 100644
--- a/utils/hwinfo/Makefile
+++ b/utils/hwinfo/Makefile
@@ -1,12 +1,12 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=hwinfo
-PKG_VERSION:=21.70
-PKG_RELEASE:=2
+PKG_VERSION:=21.71
+PKG_RELEASE:=1
 
 PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
 PKG_SOURCE_URL:=https://codeload.github.com/openSUSE/hwinfo/tar.gz/$(PKG_VERSION)?
-PKG_HASH:=bc3c4a4498e4abc32a54497ced715bbae5dfd19dd999da294bca6d69fea2db52
+PKG_HASH:=c4c573eb15cbc10103f5044b485d7e4ff941500ed559743a1c98e6a6deb0ebda
 
 PKG_LICENSE:=GPL-2.0-or-later
 PKG_MAINTAINER:=Alberto Bursi <bobafetthotmail@gmail.com>
-- 
2.25.1

